packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -211,6 +211,7 @@ export default function pluginContentPages(
                         `${docuHash(aliasedSource)}.json`,
                       );
                     },
+                    ...siteConfig.markdown,
```

Comment:
do we really want to spread instead of passing the plain "markdown" config object?

Imagine if we add `siteConfig.markdown.remarkPlugins`: we'll likely want to merge those "global remark plugins" with "plugin remark plugins" in the loader directly instead of each plugin independently?